Readme.md

```diff
@@ -103,3 +104,19 @@ In the following sections, we'll try to answer frequently asked question that we
 - **Do you help with relocation?** Yes, we do. We have an agency that would contact you to help you resolve any paperwork you might need for both our government and your own. We also provide accommodation for the first few months until you are able to find your own apartment. We have "Mews flat" in Prague and London which we manage using our own system.
 - **What is the cost of living in Prague?** You can find it and compare with other cities at https://www.numbeo.com/cost-of-living/in/Prague.
 - **How long does it take to relocate?** Unfortunately, it takes a lot of time; usually around 6 months. First, we must submit the the job posting announcing your position to the Czech Labour office, where it must be pending publicly for 30 days for potential employees. After that, the paperwork is processed, and both governments have 30-60 days to finish the procedure. Regrettably, these offices like to take their time and therefore, it is difficult to give an exact estimate about how long it takes from start to finish.
+
+<a name='media'/>
+
+### 📰 Media
+
+For 7 years we went rather unnoticed (mainly in our home base - Czech Republic) and didn't really mind as we never focused on clients in Czech Republic (only 5% of all). However, as we want to grow our tech team in Prague we need to be known and visible. And it would be pity if people didn't know about us. Year 2019 was a breakthrough in this sense in terms of media presence and public appearances.
```

Comment:
The sentence starting "However..." is IMHO irrelevant for candidates, I'd remove it.